frontend/src/buttons.scss

```diff
@@ -84,7 +84,7 @@
 }
 
 .btn.btn.btn-secondary {
-  background: none;
+  background: white;
```

Comment:
I feel like transparent may be safer choice here.

Reply:
Not 100% sure, but the secondary button has a white background on the mockups.